annotations/src/integrationTest/java/io/github/mkutz/howtotest/annotations/FriendApiTest.java

```diff
@@ -19,7 +19,7 @@ class FriendApiTest {
   @Value("http://localhost:${local.server.port}")
   String baseUrl;
 
-  @Autowired WebTestClient webClient = WebTestClient.bindToServer().baseUrl(baseUrl).build();
+  @Autowired WebTestClient webClient;
```

Comment:
The injected `webTestClient` is already bound to the test application, see [Testing With a Running Server](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.testing.spring-boot-applications.with-running-server)